### PR TITLE
feat: support aws clb in ports model

### PR DESCRIPTION
**Pull request by healthjyk**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Support aws clb in ports model.

### CODE REVIEW

File: pkg/generator/appconfiguration/generator/workload/network/ports_generator.go [hunk 1]
```diff
@@ -113,6 +116,9 @@ func (g *portsGenerator) validate() error {
 	if len(g.selector) == 0 {
 		return ErrEmptySelectors
 	}
+	if len(g.ports) == 0 {
```
Comment (SparkYuan) [inline pkg/generator/appconfiguration/generator/workload/network/ports_generator.go:119]: zero ports is allowed according to our recent discussion

Comment (healthjyk) [reply pkg/generator/appconfiguration/generator/workload/network/ports_generator.go:119]: If empty ports, there will be portsGenerator, which is done in service_generator.go
Delete validation inside the generator is not cool and make others confused, I fix it back from https://github.com/KusionStack/kusion/pull/510. I have already replied to it, and more serious review is needed. @liu-hm19

File: pkg/generator/appconfiguration/generator/workload/network/ports_generator.go [hunk 2]
```diff
@@ -186,7 +199,17 @@ func validatePorts(ports []network.Port) error {
 	return nil
 }
 
-func validatePort(port *network.Port) error {
+func validatePort(port *network.Port, portType *string) error {
```
[the remark this answers is not in the document]
Comment (healthjyk) [reply pkg/generator/appconfiguration/generator/workload/network/ports_generator.go:202]: For ports in a service, it must have the same type. portType need to get changed, so use a *.

File: pkg/generator/appconfiguration/generator/workload/network/ports_generator.go (hunk 1 above)
Comment (SparkYuan) [reply pkg/generator/appconfiguration/generator/workload/network/ports_generator.go:119]: What do you mean that portsGenerator is done in service_generator.go. Can you provide the specified code?

**Review by SparkYuan — APPROVED** [review]
lgtm

**Pull request merged**